.changesets/limit-hackney-to-1-21-0.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: patch
+type: fix
+---
+
+Hackney compatibility is limited up to 1.21.0.
```

Comment:
I believe it's more useful for users if the changelog describes the issue that is being fixed.

```suggestion
Fix an issue where the AppSignal agent fails to download during installation, by restricting the Hackney dependency to version 1.21.0 or lower.
```